bin/extract-release-notes.sh

```diff
@@ -2,22 +2,31 @@
 ###############################################################################
```

Comment:
I copied this over from HyBIG, because I also tweaked the `CHANGELOG.md` to adhere to Keep A Changelog.